src/components/z-app-header/index.tsx

```diff
@@ -204,15 +220,41 @@ export class ZAppHeader {
     if (this.menuElements.length === 0) {
       return;
     }
-
     const elements = this.menuElements;
```

Comment:
`const elements = this.menuElements as NodeListOf<HTMLZMenuElement>;`

se tipizzi in questo modo ti puoi risparmiare tutti quei (element as HTMLZMenuElement)

Reply:
Ma in realtà forse basterebbe:
- togliere tutti quei `as HTMLZMenuElement`
- togliere `const elements = this.menuElements` e usare direttamente `this.menuElements`
- mettere a riga 113: `private menuElements?: NodeListOf<HTMLZMenuElement>;`